<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuObject">
    <property name="title">
     <string>&amp;Object</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionShowMainView"/>
    <addaction name="actionShowLibraryView"/>
    <addaction name="actionShowOnlineView"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionWebSite"/>
    <addaction name="separator"/>
    <addaction name="actionSend_Feedback"/>
    <addaction name="actionReport_Bug"/>
    <addaction name="actionSend_Logs"/>
    <addaction name="actionSupport"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuObject"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowMainView"/>
   <addaction name="actionShowLibraryView"/>
   <addaction name="actionShowOnlineView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="actionWebSite">
   <property name="text">
    <string>&amp;QtApp Web Site...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About QtApp...</string>
   </property>
  </action>
  <action name="actionShowMainView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Main</string>
   </property>
   <property name="toolTip">
    <string>Show Main</string>
   </property>
  </action>
  <action name="actionShowLibraryView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Library</string>
   </property>
   <property name="toolTip">
    <string>Show Library</string>
   </property>
  </action>
  <action name="actionShowOnlineView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Online</string>
   </property>
   <property name="toolTip">
    <string>Show Online</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="text">
    <string>&amp;Technical Support...</string>
   </property>
  </action>
  <action name="actionSend_Feedback">
   <property name="text">
    <string>Send &amp;Feedback...</string>
   </property>
  </action>
  <action name="actionReport_Bug">
   <property name="text">
    <string>Report &amp;Bug...</string>
   </property>
  </action>
  <action name="actionSend_Logs">
   <property name="text">
    <string>Send &amp;Logs...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>New...</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
